benchmarks/synth-bm/Cargo.toml

```diff
@@ -1,23 +1,33 @@
 [package]
 name = "near-synth-bm"
-version = "0.0.0"
-authors = ["Near Inc <hello@nearprotocol.com>"]
-edition = "2021"
-rust-version = "1.84.0"
-repository = "https://github.com/near/nearcore"
-license = "MIT OR Apache-2.0"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+repository.workspace = true
+license.workspace = true
+publish = false
+
+[lints]
+workspace = true
+
 
 [dependencies]
-anyhow = "1.0.89"
-clap = { version = "4.5.17", features = ["derive"] }
-env_logger = "0.11.5"
-log = "0.4.22"
-near-crypto = "0.27"
-near-jsonrpc-client = "0.14.0"
-near-jsonrpc-primitives = "0.27"
-near-primitives = "0.27"
-rand = "0.8.4"
-tokio = { version = "1.40.0", features = ["full"] }
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-tracing = { version = "0.1.40", features = ["std"] }
+anyhow.workspace = true 
+clap = { workspace = true, features = ["derive"] }
+env_logger.workspace = true
+ext-near-jsonrpc-client.workspace = true
+log.workspace = true
```

Comment:
fwiw this should also be replaced by `tracing` fully in a followup of some sort.

Reply:
Opened #12911.